File: app/src/main/java/com/tory/noname/gank/bean/GankApiResult.java
package com.tory.noname.gank.bean;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class GankApiResult{

	@SerializedName("error")
	private boolean error;

	@SerializedName("results")
	private List<GankItem> results;

	public void setError(boolean error){
		this.error = error;
	}

	public boolean isError(){
		return error;
	}

	public void setResults(List<GankItem> results){
		this.results = results;
	}

	public List<GankItem> getResults(){
		return results;
	}

	@Override
 	public String toString(){
		return 
			"GankApiResult{" + 
			"error = '" + error + '\'' + 
			",results = '" + results + '\'' + 
			"}";
		}
}

File: demo/hilt/src/main/java/com/tory/module/hilt/model/GankApiResult.kt
package com.tory.module.hilt.model

import com.google.gson.annotations.SerializedName

data class GankApiResult(
    @SerializedName("error")
    var isError: Boolean = false,
    @SerializedName("results")
    var results: List<GankItem>? = null
)
